tasks/connectors/contrast/contrast.rb

```diff
@@ -0,0 +1,425 @@
+require_relative "lib/client"
+require "time"
+
+module Kenna
+  module Toolkit
+    class ContrastTask < Kenna::Toolkit::BaseTask
+      SCANNER = "Contrast"
+
+      def self.metadata
+        {
+          id: "contrast",
+          name: "Contrast",
+          description: "Extract vulnerability and library data from the Contrast platform",
+          options: [
+            { name: "contrast_host",
+              type: "hostname",
+              required: true,
+              default: nil,
+              description: "Your Contrast hostname (without protocol), e.g. app.contrastsecurity.com" },
+            { name: "contrast_use_https",
+              type: "boolean",
+              required: false,
+              default: true,
+              description: "Set to false if you would like to force an insecure HTTP connection" },
+            { name: "contrast_port",
+              type: "integer",
+              required: false,
+              default: nil,
+              description: "Your Contrast port (if on premise), e.g. 8080" },
+            { name: "contrast_api_key",
+              type: "api_key",
+              required: true,
+              default: nil,
+              description: "Your Contrast API Key, as displayed in User Settings" },
+            { name: "contrast_auth_token",
+              type: "api_key",
+              required: true,
+              default: nil,
+              description: "Your Contrast Authorization Header, which can be copied from User Settings" },
+            { name: "contrast_org_id",
+              type: "string",
+              required: true,
+              default: nil,
+              description: "Your Contrast Organization ID, as displayed in User Settings" },
+            { name: "contrast_application_tags",
+              type: "string",
+              required: false,
+              default: "",
+              description: "Filter vulnerabilities and libraries using a comma separated list of application tags" },
+            { name: "contrast_include_vulns",
+              type: "boolean",
+              required: false,
+              default: true,
+              description: "Include vulnerabilities from Contrast Assess" },
+            { name: "contrast_environments",
+              type: "string",
+              required: false,
+              default: "",
```

Comment:
for environments and severities, will nil value return all results?

Reply:
@davidaustinarcher

Reply:
Yes that is correct

Reply:
Can you note that in the read.me file?  @davidaustinarcher